src/server/views/lists/partials/translators-interpreters/translators-interpreters-results-list.njk

```diff
@@ -12,16 +12,35 @@
   <p class="govuk-body">We couldn’t find any results that match your search terms. Try changing these to search again.</p>
 {% else %}
   <h2 class="govuk-heading-l govuk-!-margin-bottom-4">{{resultsTitle}}</h2>
-  <p class="govuk-body">You should carry out your own research before choosing a service provider.</p>
 
+  {% if hasSworn.translatorsAndInterpreters or hasSworn.interpreters or hasSworn.translators %}
   <div class="govuk-warning-text">
     <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
     <strong class="govuk-warning-text__text">
       <span class="govuk-warning-text__assistive">Warning</span>
-      Translators and interpreters must be authorised to operate in some countries (known as sworn translators).
-      Check providers are qualified to carry out the service you need in the country you have selected.
+      {% if hasSworn.translatorsAndInterpreters or (hasSworn.interpreters and hasSworn.translators) %}
+        In {{ country }}, translators and interpreters must be authorised to carry out legal or official services.
+        <p>These are known as ‘sworn’, ‘certified’ or ‘court’ translators or interpreters and are listed in the results as providing official services.</p>
+      {% elif hasSworn.interpreters %}
+        In {{ country }}, interpreters must be authorised to carry out legal or official services.
+        <p>These are known as ‘sworn’, ‘certified’ or ‘court’ interpreters and are listed in the results as providing official services.</p>
+      {% elif hasSworn.translators %}
+        In {{ country }}, translators must be authorised to carry out legal or official services.
+        <p>These translators are known as ‘sworn’ or ‘certified’ translators and are listed in the results as providing official services.</p>
+      {% endif %}
```

Comment:
previously there was a catch all else block used when there are no list items flagged with sworn.  Is it intentional to leave this out?

Reply:
Fair point. Yep it's still there but structured a bit differently.

<img width="1296" alt="Screenshot 2023-04-24 at 16 37 11" src="https://user-images.githubusercontent.com/1377253/234046252-d3997d13-4efd-4948-be56-998f61f11ab9.png">